tq/transfer.go

```diff
@@ -34,17 +34,25 @@ type Transfer struct {
 	Size          int64        `json:"size"`
 	Authenticated bool         `json:"authenticated,omitempty"`
 	Actions       ActionSet    `json:"actions,omitempty"`
+	Links         ActionSet    `json:"_links,omitempty"`
 	Error         *ObjectError `json:"error,omitempty"`
 	Path          string       `json:"path,omitempty"`
 }
 
-func (t *Transfer) Rel(name string) (*Action, bool) {
-	if t.Actions == nil {
-		return nil, false
+func (t *Transfer) Rel(name string) (*Action, error) {
+	a, err := t.Actions.Get(name)
+	if a != nil || err != nil {
+		return a, err
 	}
 
-	rel, ok := t.Actions[name]
-	return rel, ok
+	if t.Links != nil {
+		a, err := t.Links.Get(name)
+		if a != nil || err != nil {
+			return a, err
+		}
+	}
+
+	return nil, nil
```

Comment:
I might pass on this one, if that's alright. Though a little more verbose, I like the flow of:

1. `t.Actions.Get()`
2. Check if we have `t.Links`
  - `t.Links.Get()`
3. Otherwise, `(nil, nil)`

If you feel strongly, though, I don't really mind either way 👍